### PR TITLE
remove non-existing includes in error page

### PR DESCRIPTION
**Pull request by goapunk**
* fixes the annoying sentry errors
* centers the text so it doesn't look totally broken

### CODE REVIEW

File: apps/core/templates/500.html [hunk 1]
```diff
@@ -1,22 +1,14 @@
 {% load static i18n %}
 <!DOCTYPE html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
-<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+<html class="no-js">
```
Comment (phillimorland) [inline apps/core/templates/500.html:3]: I don't think we need the no-js at all any more, did you find that we sac still use it without having modenizer?

Comment (goapunk) [reply apps/core/templates/500.html:3]: no, I just didn't bother to remove it as we wanted to redo the whole page anyway at some point. Should I remove it?

Comment (phillimorland) [reply apps/core/templates/500.html:3]: i'm going to go with yes, the less crap we leave in the clearer what actually need to be done whenever we next get a liqd sprint

Comment (goapunk) [reply apps/core/templates/500.html]: removed it

**Review by phillimorland — APPROVED** [review]
cool!

**Pull request merged**